appserver/tests/admin/tests/pom.xml

```diff
@@ -141,6 +141,28 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>test-app-for-ClusterITest</id>
+                        <phase>process-test-classes</phase>
+                        <goals>
+                            <goal>war</goal>
+                        </goals>
+                        <configuration>
+                            <primaryArtifact>false</primaryArtifact>
+                            <classifier>clusterit</classifier>
+                            <failOnMissingWebXml>false</failOnMissingWebXml>
```

Comment:
https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html#failOnMissingWebXml
> Starting with 3.1.0, this property defaults to false if the project depends on the Servlet 3.0 API or newer.

Since this module uses `3.3.2`, could we get rid of this explicit element then?

Reply:
The catch is that the dependency on Servlet API is in `test` scope. We could remove `<failOnMissingWebXml>` if we change the scope to `compile`. But I prefer keeping it this way, because it's explicit. If we change servlet scope to compile while all other dependencies are in `test`, it's not clear why it is like that.